mne/io/brainvision/brainvision.py

```diff
@@ -420,7 +306,7 @@ def _get_eeg_info(vhdr_fname, eog, misc, response_trig_shift):
     if settings.find('[Comment]') != -1:
         params, settings = settings.split('[Comment]')
     else:
-        params, settings = settings, str()
+        params, settings = settings, ''
```

Comment:
shouldn't this stay str() to be explicit or would that cause complications for str vs stringtype?